src/components/content/order/serviceState/restart/RestartServiceStatusAlert.tsx

```diff
@@ -0,0 +1,215 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ */
+
+import { UseMutationResult, UseQueryResult } from '@tanstack/react-query';
+import { Alert } from 'antd';
+import React from 'react';
+import {
+    ApiError,
+    DeployedService,
+    DeployedServiceDetails,
+    Response,
+    ServiceStateManagementTaskDetails,
+    VendorHostedDeployedServiceDetails,
+} from '../../../../../xpanse-api/generated';
+import { ContactDetailsShowType } from '../../../common/ocl/ContactDetailsShowType';
+import { ContactDetailsText } from '../../../common/ocl/ContactDetailsText';
+import useGetOrderableServiceDetailsQuery from '../../../deployedServices/myServices/query/useGetOrderableServiceDetailsQuery';
+import OrderSubmitResultDetails from '../../orderStatus/OrderSubmitResultDetails';
+
+function RestartServiceStatusAlert({
+    deployedService,
+    serviceStateRestartQuery,
+    closeRestartResultAlert,
+    getRestartServiceDetailsQuery,
+}: {
+    deployedService: DeployedService;
+    serviceStateRestartQuery: UseMutationResult<string, Error, DeployedService>;
+    closeRestartResultAlert: (arg: boolean) => void;
+    getRestartServiceDetailsQuery: UseQueryResult<VendorHostedDeployedServiceDetails>;
+}): React.JSX.Element {
+    const getOrderableServiceDetails = useGetOrderableServiceDetailsQuery(deployedService.serviceTemplateId);
+
+    if (
+        getRestartServiceDetailsQuery.isSuccess &&
+        [DeployedServiceDetails.serviceState.RUNNING].includes(getRestartServiceDetailsQuery.data.serviceState)
+    ) {
+        deployedService.serviceState = getRestartServiceDetailsQuery.data.serviceState;
+    }
+
+    const onClose = () => {
+        closeRestartResultAlert(true);
+    };
+
+    if (serviceStateRestartQuery.isError) {
+        let errorMessage;
+        if (
+            serviceStateRestartQuery.error instanceof ApiError &&
+            serviceStateRestartQuery.error.body &&
+            'details' in serviceStateRestartQuery.error.body
+        ) {
+            const response: Response = serviceStateRestartQuery.error.body as Response;
+            errorMessage = response.details;
+        } else {
+            errorMessage = serviceStateRestartQuery.error.message;
+        }
+        deployedService.serviceState = DeployedService.serviceState.RUNNING;
+        return (
+            <div className={'submit-alert-tip'}>
```

Comment:
no space between alert and the button below. Please make it consistent with all other alerts in the myservices UI.

![image](https://github.com/eclipse-xpanse/xpanse-ui/assets/54129659/b29faf51-be61-40ee-bc41-b8bd7788ed9f)